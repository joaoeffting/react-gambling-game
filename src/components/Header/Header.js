import React from 'react';
import styled from 'styled-components';
import unibetLogo from '../../images/unibet-logo.png';

const HeaderContainer = styled.header`
    width: 900px;
    text-align: left;
    margin: 0 auto;
    padding: 20px 0;
`;

const Logo = styled.div`
    height: 33px;
    background: url(${unibetLogo}) no-repeat;
    background-size: 200px 33px;
    text-indent: -999em;
`;

export const Header = () => {
    return (
        <HeaderContainer>
            <Logo>Unibet</Logo>
        </HeaderContainer>
    )
}