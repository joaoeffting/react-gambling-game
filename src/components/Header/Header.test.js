import React from 'react';
import { Header } from './Header';
import renderer from 'react-test-renderer';

test('Header match snapshot', () => {
  const component = renderer.create(
    <Header />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});