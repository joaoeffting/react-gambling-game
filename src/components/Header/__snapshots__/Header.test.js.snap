// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match snapshot 1`] = `
<header
  className="sc-bdVaJa ezWSqq"
>
  <div
    className="sc-bwzfXH gRZoIT"
  >
    Unibet
  </div>
</header>
`;
